src/services/map/ol.synchronizer.ts

```diff
@@ -30,7 +30,7 @@ export class OlSynchronizer {
       })
 
       if (newContext.layers) {
-        Openlayers.reorderLayers(map, newContext.layers)
+        Openlayers.reorderLayers(map, [...(newContext.layers ?? [])].reverse())
```

Comment:
I don't get it, `context.layers` should be in the correct order so why using a reverse here ?